spec/models/order_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe Order, type: :model do
+  describe '#valid?' do
+    it {should belong_to(:user)}
+    it {should have_many(:cart_items)}
+    it {should validate_presence_of(:address)}
+
+    context 'regarding the cart' do
+      it "must have a cart" do
+        order = build(:order)
+
+        expect(order.valid?).to be false
```

Comment:
```suggestion
        expect(order).to be_false
```
Só para mostrar essa outra forma

Reply:
existe também `expect(order).not_to be_valid`. mas não recomendo vocês trocarem, o erro é mais fácil de ler do jeito original que vocês escreveram

Reply:
esse que a Agatha mostrou é o jeito certo, eu escrevi errado ai, foi mal

Reply:
Boa, não sabia disso.
Inclusive: "While be true‘s name literally describes what it asserts, be_true‘s name is arguably ambiguous. To prevent future confusion, be_true will be renamed be_truthy and be_false will become be_falsey in the upcoming RSpec 3.0 release." - [Janeiro de 2014](https://bengribaudo.com/blog/2014/01/17/2674/rspec-be-true-be_true-are-different)

Reply:
Infelizmente esse `be_falsey` não funcionou ☹️